packages/compass-e2e-tests/tests/ttfq.test.js

```diff
@@ -0,0 +1,32 @@
+// @ts-check
+const { expect } = require('chai');
+const { beforeTests, afterTests } = require('../helpers/compass');
+
+describe('Time to first query', function () {
+  this.timeout(1000 * 60 * 1);
+
+  let compass;
+
+  it('can open compass and connect to a database', async function () {
+    // start compass inside the test so that the time is measured together
+    ({ compass } = await beforeTests(false));
+
+    await compass.client.connectWithConnectionString(
+      'mongodb://localhost:27018/test'
+    );
+
+    const result = await compass.client.shellEval(
+      'db.runCommand({ connectionStatus: 1 })',
+      true
+    );
+
+    expect(result).to.have.property('ok', 1);
+  });
```

Comment:
I would actually expect this to submit a query tbh, shell might be interactive much earlier than the actual CRUD screen

Reply:
I agree. I'll tackle it in the next issue as I'm clearly going to have to load in a fixture of collections for the tests and then we'll have something they can query. But please also chime in on the relevant slack threads.

Reply:
My bad, I saw the PR is merged and assumed we are done with it, but I see how that the conversation is still going and the ticket is "in progress" 👍